### PR TITLE
Yarn instead of npm

### LINKED ISSUE #237: Incompatibility with npm
I'm neither able to install nor build modulizer using npm.
Though it totally works with yarn.

### PR DESCRIPTION
**Pull request by web-devel**
Fixes #237

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,17 +13,17 @@
   },
   "scripts": {
     "build": "tsc",
-    "test": "npm run build && mocha && npm run lint",
+    "test": "yarn run build && mocha && yarn run lint",
```
Comment (stramel) [inline package.json]: you can do `yarn build && mocha && yarn lint`. `npm` requires the use of `run` keyword before the script name.

File: README.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ window.customElements.define(MyElement.is, MyElement);
 Install polymer-modulizer from npm:
 
 ```
-npm install -g polymer-modulizer
+yarn install -g polymer-modulizer
```
Comment (stramel) [inline README.md]: I think this is also supposed to be `yarn global add polymer-modulizer`

**Comment by justinfagnani** [comment]
We're not requiring Yarn to install modulizer, so any fix to #237 should actually fix whatever problem there is with that particular npm installation.

**Comment by FredKSchott** [comment]
Yea, I'd go a step further and say these should remain `npm run`. 

While we recommend yarn for a lot of things, we still also support `npm`. `npm run` is guaranteed to work for anyone (it comes with Node) while `yarn run` would fail without yarn.

Lets fix the npm issue itself instead!

**Comment by web-devel** [comment]
Thanks for the clarification, it make sense. I think it probably might be better to change `yarn install` →`npm install` in readme then.

**Pull request closed without merging**